templates/al2023/provisioners/install-nodeadm.sh

```diff
@@ -13,6 +13,10 @@ sudo nerdctl run \
   public.ecr.aws/eks-distro-build-tooling/golang:1.21 \
   make build
 
+# cleanup images and networks used for nerdctl
+sudo nerdctl image prune --all --force
+sudo nerdctl network rm $(sudo nerdctl network list -q)
```

Comment:
Could we use the `--network=none` flag to `nerdctl run` to avoid creating this in the first place?

Reply:
that'll break the ability to pull packages in the build, but maybe using the `host` network wont create an additional interface. need to check